NeuroID/src/main/java/com/neuroid/tracker/service/NIDJobServiceManager.kt

```diff
@@ -144,15 +143,15 @@ internal class NIDJobServiceManager(
      */
     private fun createGyroCadenceServer(): Job {
         return CoroutineScope(dispatcher).launch {
-            while (NeuroID.isSDKStarted && NeuroID.captureGyroCadence) {
-                delay(NeuroID.GYRO_SAMPLE_INTERVAL)
+            while (NeuroID.isSDKStarted && NeuroID.nidSDKConfig.gyroAccelCadence) {
+                delay(NeuroID.nidSDKConfig.gyroAccelCadenceTime * 1000L)
```

Comment:
This would be 200L * 1000L? Isn't that something like 200 seconds?

Reply:
removed the multiplier. will take the value directly from the config.